src/Controllers/UserStoryController.php

```
@@ -495,6 +513,45 @@ public function update($id): void
+     * @param int $id User story primary key (from route parameter)
+     */
+    public function editData($id): void
+    {
+        $user  = $this->auth->user();
+        $story = UserStory::findById($this->db, (int) $id);
+        if ($story === null) {
+            $this->response->json(['status' => 'error', 'message' => 'Story not found'], 404);
+            return;
+        }
+
+        $project = ProjectPolicy::findViewableProject($this->db, $user, (int) $story['project_id']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Security\ProjectPolicy' in method 'editData'. (undefined)

(StaticAccess)